diff --git a/CMakeLists.txt b/CMakeLists.txt
index ca2639961..10e2f9a31 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -108,7 +108,7 @@ sbom_generate(
         SUPPLIER_URL "http://github.com/sithlord48/ff7tk"
 )
 #Qt Info to get deploy app
-find_package(Qt6 ${REQUIRED_QT_VERSION} REQUIRED COMPONENTS Core)
+find_package(Qt6 ${REQUIRED_QT_VERSION} REQUIRED COMPONENTS Core LinguistTools)
 get_target_property(qmake_executable Qt6::qmake IMPORTED_LOCATION)
 get_filename_component(_qt_bin_dir "${qmake_executable}" DIRECTORY)
 if(WIN32 OR APPLE)
diff --git a/translations/CMakeLists.txt b/translations/CMakeLists.txt
index c06f03649..f96ff6012 100644
--- a/translations/CMakeLists.txt
+++ b/translations/CMakeLists.txt
@@ -3,7 +3,6 @@
 
 set_directory_properties(PROPERTIES CLEAN_NO_CUSTOM 1)
 option(CLEAN_TRS "Clean Obsolete translations from tr files" FALSE)
-find_package(Qt6LinguistTools)
 
 set(ff7tk_TRS
     ff7tk_de.ts
